docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,108 +1,75 @@
-in visualizations and Canvas workpads.
-With GeoJSON Upload, you can upload a file up to 50 MB.
-
-[float]
-[[add-data-tutorial-kibana]]
-===  Index metrics, log, security, and application data
 
-The built-in data tutorials can help you quickly get up and running with
-metrics data, log analytics, security events, and application data.
-These tutorials walk you through installing and configuring a
-Beats data shipper to periodically collect and send data to {es}.
-You can then use the pre-built dashboards to explore and analyze the data.
+[discrete]
+=== Custom options for loading your data
 
-You access the tutorials from the home page.
-If a tutorial doesn’t exist for your data, go to the {beats-ref}/beats-reference.html[Beats overview]
-to learn about other data shippers in the Beats family.
+If the built-in options don't work for you, you can index your
+data into Elasticsearch with {ref}/getting-started-index.html[REST APIs]
+or https://www.elastic.co/guide/en/elasticsearch/client/index.html[client libraries].
+After you add your data, you're required to create an <<index-patterns,index pattern>> to tell
+{kib} where to find the data.
 
-[role="screenshot"]
-image::images/add-data-tutorials.png[Add Data tutorials]
-
-
-[float]
-[[connect-to-es]]
-=== Connect with {es} indices
-
-To visualize data in existing {es} indices, you must
-create an index pattern that matches the names of the indices that you want to explore.
-When you add data with the File Data Visualizer, GeoJSON Upload feature,
-or built-in tutorial, an index pattern is created for you.
-
-. Open the main menu, then click *Stack Management > Index Patterns*.
-
-. Click *Create index pattern*.
-
-. Specify an index pattern that matches the name of one or more of your Elasticsearch indices.
-+
-For example, an index pattern can point to your Apache data from yesterday,
-`filebeat-apache-4-3-2022`, or any index that matches the pattern, `filebeat-*`.
-Using a wildcard is the more popular approach.
-
-
-. Click *Next Step*, and then select the index field that contains the timestamp you want to use to perform time-based
-comparisons.
-+
-Kibana reads the index mapping and lists all fields that contain a timestamp. If your
-index doesn't have time-based data, choose *I don't want to use the time filter*.
-+
-You must select a time field to use global time filters on your dashboards.
-
-. Click *Create index pattern*.
-+
-{kib} is now configured to access your {es} indices.
-You’ll see a list of fields configured for the matching index.
-You can designate your index pattern as the default by clicking the star icon on this page.
-+
-When searching in *Discover* and creating visualizations, you choose a pattern
-from the index pattern menu to specify the {es} indices that contain the data you want to explore.
+* To add data for Elastic Observability, see {observability-guide}/add-observability-data.html[Send data to Elasticsearch].
```

Comment:
Probably another discussion, but a common naming convention and framework for ingestion "How to's" might make it easier to consume for our end users.